src/modules/billing/controllers/invoices.js

```diff
@@ -14,4 +14,25 @@ const patchInvoice = async request => {
   }
 };
 
+const resetIsFlaggedForRebilling = async request => {
+  try {
+    const { batchId } = request.params;
+
+    return await invoiceService.resetIsFlaggedForRebilling(batchId);
+  } catch (err) {
+    return mapErrorResponse(err);
+  }
+};
+
+const resetIsFlaggedForRebillingByInvoiceId = async request => {
+  try {
+    const { originalInvoiceId } = request.params;
```

Comment:
this will just be the invoiceId instead of originalInvoiceId because within this context we only receive on id